process/block/postprocess/feeHandler.go

```diff
@@ -0,0 +1,53 @@
+package postprocess
```

Comment:
struct not tested. Maybe add a task to improve code coverage on this struct, intermediateResults.go and preprocess subpackage (we have there some files with poor code coverage)

Reply:
agreed. Test will be added for economics V2. added task.

Reply:
:+1: